README.md

```diff
@@ -12,3 +12,8 @@ An open source data gateway.
 - Video tutorial 5-10min of using Stargate (serverless)
 - CRUD usage of Stargate
 - Discord Bot (bio, social links, timezone)
+
+## Reproducing locally-
```

Comment:
What is the `-`

Reply:
Um it's like a heading referring to things(like colons/dashes before points)
It can be removed, if these are not needed(I think these shouldn't be used here since, consistently we're not using `-` after headings.)

Reply:
resolved by 9aec906